src/service/world-api-adapter.js

```diff
@@ -141,7 +141,7 @@ module.exports = function (config) {
             }
 
             // account and project go in the body, not in the url
-            $.extend(params, _pick(serviceOptions, ['account', 'project', 'group']));
+            $.extend(params, _pick(serviceOptions, ['account', 'project']));
```

Comment:
This should not be committed, I removed this one time to make some world api adapter test pass and I forgot to revert it back